trailing_spaces.py

```diff
@@ -427,6 +482,33 @@ def on_pre_save(self, view):
         if ts_settings.get("trailing_spaces_trim_on_save"):
             view.run_command("delete_trailing_spaces")
 
+    def on_close(self, view):
+        # untrack
+        active_views.pop(view.id(), None)
+
+    def update_view_on_scroll(self, view):
+        # panel views don't trigger on_close but are also not valid anymore
+        # after being hidden, so try to detect these cases here
+        if view.size() == 0 and not view.file_name():
```

Comment:
Wouldn't it make sense to put this check in `is_view_visible()` also?
I think it does since that's what it essentially does and also this duplicates pop() call.

You can just say whether you agree, don't have to change the code. I will update your PR is you agree as I've made some tiny style fixes already.

Reply:
Absolutely, feel free to change whatever you like.